general/development/process/testing/guide.md

```diff
@@ -0,0 +1,25 @@
+---
+title: Testing instructions guide
+tags:
+  - ' Processes'
+---
+This page has suggestions for developers on how to write good testing instructions for the weekly [testing of integrated issues](./integrated-issues).
+
+We recommend that you:
+
+1. Number the steps in your test, and make use of sub-lists.
+2. Only put one action (preparation or validation) on each line - A step should only define a unique operation.
+3. Promote test validations - **Confirm**, **Verify** or **Ensure** - should be in **bold** so that they are easily identifiable.
+4. Make use of the [Jira Markdown formatting](https://tracker.moodle.org/secure/WikiRendererHelpAction.jspa?section=all).
+
+In addition, the following items may be included:
+
+1. Setup requirements. If so, explain them with detailed information and/or provide a link to the documentation e.g. <https://docs.moodle.org/en/OAuth_2_services>.
```

Comment:
```suggestion
1. Setup requirements. If so, explain them with detailed information and/or provide a link to the documentation, for example <https://docs.moodle.org/en/OAuth_2_services>.
```

Several of the writing guides I've looked at recommend not using `e.g.` but using `for example`.
With `i.e.` they recommend `that is`.

- https://docs.microsoft.com/en-us/style-guide/a-z-word-list-term-collections/e/eg
- https://docs.microsoft.com/en-us/style-guide/a-z-word-list-term-collections/i/ie

Reply:
Thanks for the clarification. I wasn't unsure about that and, as I saw several references to e.g. in our current devdocs, I thought it was ok. Now I know the best wording for them! :-)